app/src/main/java/nl/tudelft/cs4160/identitychain/peers/Peers.kt

```diff
@@ -0,0 +1,42 @@
+package nl.tudelft.cs4160.identitychain.peers
+
+import com.google.protobuf.ByteString
+import nl.tudelft.cs4160.identitychain.message.ChainService
+import java.util.*
+
+/**
+ * The minimal amount of information required to start a grpc connection to a peer.
+ * The port, for now hard coded since every grpc server runs on 8001
```

Comment:
eh, 8001 here and 8080 below don't match up?

Reply:
True, I was confused back then with the port they run on in the test.
They do actually run on 8080. I'll update the comment